.github/workflows/test.yml

```diff
@@ -146,7 +146,7 @@ jobs:
         working-directory: control-plane
         run: |
           mkdir -p $HOME/bin
-          wget https://github.com/ndhanushkodi/binaries/releases/download/v4.1oss/consul -O consulbin && \
+          wget https://github.com/ndhanushkodi/binaries/releases/download/v6oss/consul -O consulbin && \
```

Comment:
This is based of of 1.13.x branch of consul

Reply:
Should we wait for core release before merging?

Reply:
ya that works or I can make an explicit revert change to switch back to releases.hashicorp.com right after core release-- i think if it takes them past the early afternoon, then I'll merge this for today and make the explicit revert change! that way we can have our unit tests running against 1.13 on main as we're merging other k8s PRs too

Reply:
theyve released! So i'll update this PR